sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -122,24 +129,24 @@ object AnalysisContext {
  */
 class Analyzer(
     override val catalogManager: CatalogManager,
+    v1SessionCatalog: SessionCatalog,
```

Comment:
I renamed `catalog` to `v1SessionCatalog` in `Analyzer` to be explicit. Please let me know if this is not desired.